### DISCUSSION TITLE
80 columns using 4x8 font

### DISCUSSION THREAD

**Discussion by erlendnagel** [paulscottrobson/neo6502-firmware#392] [Ideas]
I've created a small #library that uses a tileset (top left quarter filled) with 4x8 characters to produce 80 columns. This could really do with some firmware support though, because this way I cannot combine 80 columns with tiles and I get the black border around the characters. It would be ideal if the character maps could be part of the firmware (not using RAM) and a method existed to just plot the 4x8 character that the library could use (col80.printchar / col80.println etc.).

Any views on whether this is worthwhile doing?

**Comment by IanWitham** [comment]
Tali Forth block editor really needs 70-80 column display. A way to change screen modes would be good. Default to current mode so that existing programs aren't broken

**Comment by paulscottrobson** [comment]
Not possible ; there's one screen mode.

**Reply by IanWitham** [reply]
Is there a technical reason this couldn't be done, if I wanted to play with the concept in my own fork? cheers

**Comment by paulscottrobson** [comment]
As you suggest, none at all. 4x8 can be a little difficult to read, but it's basically just a bitmap so it would work. Adding it to the core is relatively straightforward, it would just be an addition to the current API. Actually making the screen wider (e.g. 640px) is technically possible but not that easy, and I'm not sure PicoDVI copes well with switches.
